src/source/Ice/IceAgent.c

```diff
@@ -1902,7 +1902,7 @@ STATUS iceCandidateSerialize(PIceCandidate pIceCandidate, PCHAR pOutputData, PUI
                                  pIceCandidate->ipAddress.address[1],
                                  pIceCandidate->ipAddress.address[2],
                                  pIceCandidate->ipAddress.address[3],
-                                 (UINT16) getInt16(pIceCandidate->ipAddress.port),
+                                 (UINT16) SWAP_INT16(pIceCandidate->ipAddress.port),
```

Comment:
Nice!

Reply:
Maybe not... This might not work on LE machine properly... Needs more though

Reply:
Agreed. As I have limited insight into the internals.  Please suggest code changes and I'd happy to update this PR.

Reply:
can you change it back to getInt16? it should evaluate to SWAP_INT16 in LE machine.

Reply:
This change has been reverted. The cause was in my code, not initializing the library.